src/main/kotlin/io/github/mojira/arisa/modules/Helpers.kt

```diff
@@ -116,3 +118,112 @@ fun assertAfter(instant1: Instant, instant2: Instant) = if (instant1.isAfter(ins
 } else {
     OperationNotNeededModuleResponse.left()
 }
+
+fun concatLinkName(vararg arguments: String): Array<String> {
+    val args = arguments.flatMap { s ->
+        s.split(',').filter {
+            it != ""
+        }
+    }.toTypedArray()
+    fun check(str: String): Boolean {
```

Comment:
I would maybe move this function out of the other function, you can also turn it into a extension function like

fun String.checkIfLinkNameRegexMatches() or something like that(instead of str you use this)